### PR TITLE
Replace non-ASCII characters in gameui debug display code

### PR DESCRIPTION
**Pull request by osjc**
Various fixes to improve compliance of the code with the coding standards.

### CODE REVIEW

**Comment by osjc** [comment]
Currently a few of non-ASCII characters need to be removed from the area of code where I am working.

**Comment by SmallJoker** [comment]
Which coding standards exactly?

**Comment by DS-Minetest** [comment]
https://dev.minetest.net/Code_style_guidelines#Miscellaneous
>Do not use characters in C++ files from outside the ASCII character set. 

I didn't know that `°` isn't ASCII.

**Comment by ClobberXD** [comment]
> Which coding standards exactly?

It looks like this PR is WIP.

**Comment by osjc** [comment]
Yes, there is a lot more coming. For example the nitpicks from the #8800 and #8793 PRs.

File: src/client/gameui.cpp [hunk 1]
```diff
@@ -129,9 +129,9 @@ void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_
 			<< "pos: (" << (player_position.X / BS)
 			<< ", " << (player_position.Y / BS)
 			<< ", " << (player_position.Z / BS)
-			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
+			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "\xC2\xB0 "
```
Comment (sfan5) [inline src/client/gameui.cpp]: isn't this utf-8 specific?
shouldn't this be an `\uabcd` style escape?

Comment (osjc) [reply src/client/gameui.cpp]: This is exactly equivalent to what is in the original code. The original code contains the utf-8 character spelled out directly, I just converted that into C escapes to bring it down to ASCII. "\uabcd" is a "wide char string" which would prompt for much more intrusive code change, at least here and I am not sure whether elsewhere too.

**Comment by osjc** [comment]
Unforseen life circumstances caused my attention to be diverted to other things for longer than acceptable. Converted this PR into a single topic PR. The remaining coding standard fixes will be submitted as a different PR once I get them ready.

**Comment by paramat** [comment]
See #9828, seems this PR was not necessary?

**Pull request merged**